mizar/obj/droplet.py

```diff
@@ -21,6 +21,8 @@
 
 import logging
 import json
+import netifaces
```

Comment:
remove netifaces, we use environment variable as the only source for interface name for now